src/main/java/touro/subwaymap/SubwayService.java

```diff
@@ -0,0 +1,88 @@
+package touro.subwaymap;
+
+import java.io.IOException;
+import java.util.*;
+import static java.util.Collections.*;
+import static touro.subwaymap.SubwayStations.*;
+
+public class SubwayService {
+    //clean, which belong in methods? public vs private?
```

Comment:
if a variable is only used in a method, it should be declared in that method. Not as a member variable.